src/client/components/UserHeader.js

```diff
@@ -85,6 +86,11 @@ const UserHeader = ({
         </div>
         <div className="UserHeader__handle-rank-container">
           <div className="UserHeader__row UserHeader__handle">@{handle}</div>
+          {!isSameUser && isFollowingYou && (
+            <div className="UserHeader__followsyou">
```

Comment:
Correct me if I'm wrong but, I think this should be `UserHeader--follows-you`

As @Sekhmet mentioned before we use http://getbem.com/introduction/ for our class names

Reply:
Changed to `UserHeader--follows-you`